test cases/cmake/24 mixing languages/meson.build

```
@@ -7,6 +7,11 @@ if not add_languages('objc', required : false)
   error('MESON_SKIP_TEST: No ObjC compiler')
 endif
 
+objc = meson.get_compiler('objc')
+if objc.get_define('_MSC_VER') != ''
+  error('MESON_SKIP_TEST: fixme: cmake must get the correct compiler when mixing languages')
+endif
+
```

If you want, once the fixes in my comment are applied, you can document why it actually doesn't work:

```suggestion
objc = meson.get_compiler('objc')
c = meson.get_compiler('c')
if c.get_argument_syntax() != objc.get_argument_syntax()
  error('MESON_SKIP_TEST: cmake cannot mix compiler types on windows')
endif
```